apps/application-system/api/src/app/modules/application/files/utils/aws.ts

```diff
@@ -0,0 +1,55 @@
+import * as AWS from 'aws-sdk'
+import { environment } from '../../../../../environments'
+
+const s3 = new AWS.S3()
+const oneMinute = 60
+const bucket = environment.fsS3Bucket ?? 'development-legal-residence-change'
```

Comment:
Do we want to hardcode the default here? Don't we want this to fail on prod for example if the env variable is missing?